lib/Steps/Integrations/ReactNative.ts

```diff
@@ -420,6 +420,7 @@ The snippet will create a button that, when tapped, sends a test event to Sentry
       {
         shellPath: '/bin/sh',
         shellScript: `
+source "../node_modules/react-native/scripts/xcode/with-environment.sh"
```

Comment:
Because we support RN from 0.65.0 and newer we need to check if the file exists first.

```suggestion
WITH_ENVIRONMENT="../node_modules/react-native/scripts/xcode/with-environment.sh"
if [ -f "$WITH_ENVIRONMENT" ]; then
    source "$WITH_ENVIRONMENT"
fi 
```